### PR TITLE
Add new Tomcat metrics dashboard in Grafana

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
Add new tomcat dashboard with session metrics:
Screenshot from 2019-09-18 18-10-24

### CODE REVIEW

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 1]
```diff
@@ -6241,6 +6241,581 @@ data:
       "uid": "sL0Klq_mz",
       "version": 1
     }
+    che-tomcat.json: |-
```
Comment (skabashnyuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: can you add panels to Che Server dashboard?

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 2]
```diff
@@ -6241,6 +6241,581 @@ data:
...
+            "panels": [],
+            "title": "Sessions",
+            "type": "row"
+          },
+          {
+            "aliasColors": {},
+            "bars": false,
+            "dashLength": 10,
+            "dashes": false,
+            "datasource": "che",
```
Comment (skabashnyuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: ds as variable

**Comment by skabashnyuk** [comment]
Can you add some screenshots?

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml (hunk 1 above)
Comment (mkuznyetsov) [reply deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: ok

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml (hunk 2 above)
Comment (mkuznyetsov) [reply deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: fixed

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 3]
```diff
@@ -6208,7 +6738,7 @@ data:
         ]
       },
       "time": {
-        "from": "now-24h",
+        "from": "now-1h",
```
Comment (skabashnyuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: can you remove this change?

**Review by skabashnyuk — APPROVED** [review]
overall look good

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml (hunk 3 above)
Comment (mkuznyetsov) [reply deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: ok

**Pull request merged**